spec/controllers/coronavirus_form/check_answers_controller_spec.rb

```diff
@@ -41,6 +41,10 @@
         ENV["HEROKU_APP_NAME"] = "coronavirus-form-preview"
       end
 
+      after do
+        ENV["HEROKU_APP_NAME"] = nil
+      end
+
```

Comment:
Hi Bill, i'm not really familiar with the use of  `after do` does it just execute that bit of code when the `context "when running as a Heroku preview app" do ` ends?